languages/csharp/concepts/time/about.md

```diff
@@ -1 +1,42 @@
-TODO: add information on time concept
+Although this exercise investigates the concept of `time` in practice you rarely deal with times on their own. They are almost always dealt with in conjunction with dates. There is no specific separate _time_ type, only [`DateTime`][date-time].
```

Comment:
> Although this exercise 

Just for future reference, we should be removing all the exercise references from the concept documents as they are to be standalone documents.